### DISCUSSION TITLE
Decoding signal from wireless doorbell button

### DISCUSSION THREAD

**Discussion by RoEdAl** [merbanan/rtl_433#2773] [General]
I am trying to decode [this](https://triq.org/pdv/#AAB10600800108004001CC05F02714848090A0A090A0A0A09090A0A0B090A0A094A090A0A090A0A0A09090A0A0B090A0A094A090A0A090A0A0A09090A0A0B08555) signal from remote doorbell button.

Decoding seems to me rather simple:

* First frame preamble length: 128 HIGH, second frame preamble length: 64 HIGH, third frame incomplete.
* **short** bit (`0`) - length: 128 **LOW**.
* **long** bit (`1`) - length: ~200 HIGH.
* **sync** (between bits) - length: 64 **HIGH**.

Is it possible to use *flex* decoder (and specify options via `-X` parameter) for this signal or maybe there is another ready-to-use decoder able to decode this signal?

**Comment by zuckschwerdt** [comment]
I'd use 64 µs PCM and use the full code as it's only a single fixed code anyway (doorbell).
